app/pages/home/home.tsx

```diff
@@ -137,7 +137,7 @@ export const Home: FC = () => {
 
   return (
     <>
-      <ReceiveStxModal isOpen={receiveModalOpen} />
+      {receiveModalOpen && <ReceiveStxModal isOpen={receiveModalOpen} />}
```

Comment:
this will cause the modal to no longer smoothly animate.

Reply:
Ffs I'd written a reply, resovled before commenting and it deleted it. 

Tl;dr that this causes the components to render in the bg, also triggering the "listen for ledger" functionality, causing it to always poll. I can do the conditional in the component though

Reply:
> Tl;dr that this causes the components to render in the bg, also triggering the "listen for ledger" functionality, causing it to always poll. I can do the conditional in the component though

yeah figured as much! I find that using the `isOpen` param within the modal itself to hide/show children is helpful when you might not want certain effects to happen while it's in the background, while still leaving the smooth animations in place